src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -2917,7 +2917,7 @@ void java_lang_LiveStackFrameInfo::set_mode(oop element, int value) {
 }
 
 
-// java_lang_reflect_AccessibleObject
+// java_lang_AccessibleObject
```

Comment:
Spurious change?

Reply:
No. I was like that before I made any hotspot change. I'd like to leave what it was -- because we're making only minimal hotspot changes.

Reply:
Ah, nvm, it only shows up in the diff of the commit.